src/libs/ReportUtils.js

```diff
@@ -1970,6 +1970,28 @@ function getChatByParticipants(newParticipantList) {
     });
 }
 
+/**
+ * Attempts to find a report in onyx with the provided email list of participants. Does not include threads
+ * This is temporary function while Migrating from PersonalDetails to PersonalDetailsList
+ *
+ * @deprecated - use getChatByParticipants()
+ *
+ * @param {Array} participantsloginList
+ * @returns {Array|undefined}
+ */
+function getChatByParticipantsByLoginList(participantsloginList) {
+    participantsloginList.sort();
+    return _.find(allReports, (report) => {
+        // If the report has been deleted, or there are no participants (like an empty #admins room) then skip it
+        if (!report || !report.participants || isThread(report)) {
```

Comment:
Could we add `!isUserCreatedPolicyRoom(report)` to this early return too?

Reply:
I would just keep it as it is as we're partially reverting this commit https://github.com/Expensify/App/commit/73cdf02a8d0487d70aad6df6d365168f0682b4da#diff-65c096044d5f69b35bcdec14c99c4fda4580759df9b1a7c36650d58eea276f1dR1890